docs: add node version requirements to installation (#87)
Nutlope/aicommits: README.md

---

## Installation and Usage
Before installing make sure you have the current LTS (v14~18) of Node.js. You can check your node version using ```bash node -v```. To update to latest node version run
```bash
sudo npm cache clean -f
sudo npm install -g n
sudo n stable

> ```

Reviewer: 1. `sudo` should not be used because it will install packages with root privileges: https://github.com/tj/n#:~:text=lib%2C%20and%20share.-,To%20avoid%20requiring%20sudo,include%20%2Fusr%2Flocal%2Fshare,-If%20npm%20is

2. The npm cache clean step should be unnecessary. Each npm version will have it's own cache anyway.

3. Personally, I use nvm or pnpm so I don't think it's necessary to include these specific steps. We should just say the minimum Node.js version is LTS v14.

Author: Okay I just did a quick search on how to update node.js and found that on stack exchange

Author: I removed the installation commands and changed the wording of the minimum Node section version section

---

## Installation and Usage

> Before installing make sure you have at least LTS v14 or greater of Node.js. You can check your node version using ```bash node -v```.

Reviewer: ```suggestion
The minimum supported version of Node.js is v14. You can check your Node.js version with `node --version`.
```

FYI you use single backticks for inline code. And the `bash` part is unnecessary—it can be run in any shell.

Author: Thanks, I didn't know you could use single backticks. Those changes make more sense for the section.
